core/ui/common/qam/powertools_menu.gd

```diff
@@ -0,0 +1,316 @@
+extends VBoxContainer
+
+var Platform := load("res://core/global/platform.tres")
```

Comment:
Same as mentioned in another comment, this shadows the `Platform` class, and should add `as Platform` to infer the correct type.

Reply:
This was not longer referenced/used. removed.